site/content/blog/2022-09-01-whats-new-in-svelte-september-2022.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "What's new in Svelte: September 2022"
+description: "Migrating to SvelteKit's new filesystem-based router"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With the redesign of SvelteKit's filesystem-based router merging early last month, there's lots to cover this month - from the [migration script](https://github.com/sveltejs/kit/discussions/5774) to a number of new blog posts, videos and tutorials.
+
+But the new routing isn't the only new feature in SvelteKit...
+
+## What's new in SvelteKit
+- `Link` is now supported as an HTTP header and works out of the box with Cloudflare's [Automatic Early Hints](https://github.com/sveltejs/kit/issues/5455) (**1.0.0-next.405**, [PR](https://github.com/sveltejs/kit/pull/5735))
+- Svelte components are now lazy loaded to reenable no-ssr use cases (**1.0.0-next.412**, [PR](https://github.com/sveltejs/kit/pull/5930))
```

Comment:
This was a regression of the new routing system - it worked before and now does again, so I would remove it from this list